io.github.letslearn-app.yaml

```diff
@@ -0,0 +1,56 @@
+app-id: io.github.letslearn.letslearn-app
```

Comment:
1. you have to rename the manifest too
2. this still doesn't meet requirements. the org is `letslearn-app` so the 3rd component should be `letslearn_app`